api/models.py

```diff
@@ -37,7 +37,7 @@ def save(self, *args, **kwargs):
                 self.settings = resources.meeting_json
             elif self.collection_type == 'dataset':
                 self.settings = resources.dataset_json
-        super(Collection, self).save(args, kwargs)
+        super(Collection, self).save(*args, **kwargs)
```

Comment:
i think it can just be `super().save` too

Reply:
I'll test it